UNITS.md

```diff
@@ -0,0 +1,201 @@
+# All quantities
+ - [AbsorbedDose](#absorbeddose)
+ - [Acceleration](#acceleration)
+ - [AmountOfSubstance](#amountofsubstance)
+ - [Angle](#angle)
+ - [AngularVelocity](#angularvelocity)
+ - [Area](#area)
+ - [Capacitance](#capacitance)
+ - [CatalyticActivity](#catalyticactivity)
+ - [ElectricCharge](#electriccharge)
+ - [ElectricCurrent](#electriccurrent)
+ - [ElectricalConductance](#electricalconductance)
+ - [Energy](#energy)
+ - [Force](#force)
+ - [Frequency](#frequency)
+ - [Illuminance](#illuminance)
+ - [Inductance](#inductance)
+ - [Length](#length)
+ - [LuminousIntensity](#luminousintensity)
+ - [MagneticFlux](#magneticflux)
+ - [MagneticFluxDensity](#magneticfluxdensity)
+ - [Mass](#mass)
+ - [Power](#power)
+ - [Pressure](#pressure)
+ - [Resistance](#resistance)
+ - [Temperature](#temperature)
+ - [Time](#time)
+ - [Velocity](#velocity)
+ - [Voltage](#voltage)
+ - [Volume](#volume)
+
+
+# All Units
+
+## AbsorbedDose
+ - gray
+
+## Acceleration
+ - meterPerSecondSquared
+
+## AmountOfSubstance
+ - mole
+
+## Angle
+ - degree
+ - radian
+
+## AngularVelocity
+ - degreePerSecond
+ - radianPerSecond
+ - rpm
+
+## Area
+ - squareMeter
+
+## Capacitance
+ - farad
+
+## CatalyticActivity
+ - katal
+
+## ElectricCharge
+ - coulomb
+
+## ElectricCurrent
+ - ampere
+ - attoampere
+ - exaampere
+ - femtoampere
+ - gigaampere
+ - kiloampere
+ - megaampere
+ - microampere
+ - milliampere
+ - nanoampere
+ - petaampere
+ - picoampere
+ - teraampere
+ - yoctoampere
+ - yottaampere
+ - zeptoampere
+ - zettaampere
+
+## ElectricalConductance
+ - siemens
+
+## Energy
+ - joule
+
+## Force
+ - newton
+
+## Frequency
+ - becquerel
+ - herz
+
+## Illuminance
+ - lux
+
+## Inductance
+ - henry
+
+## Length
+ - attometer
+ - exameter
+ - femtometer
+ - gigameter
+ - kilometer
+ - megameter
+ - meter
+ - micrometer
+ - millimeter
+ - nanometer
+ - nauticalMile
+ - petameter
+ - picometer
+ - terameter
+ - yoctometer
+ - yottameter
+ - zeptometer
+ - zettameter
+
+## LuminousIntensity
+ - candela
+
+## MagneticFlux
+ - weber
+
+## MagneticFluxDensity
+ - tesla
+
+## Mass
+ - attogram
+ - exagram
+ - femtogram
+ - gigagram
+ - gram
+ - kilogram
+ - lispund
+ - megagram
+ - microgram
+ - milligram
+ - nanogram
+ - petagram
+ - picogram
+ - skeppund
+ - teragram
+ - yoctogram
+ - yottagram
+ - zeptogram
+ - zettagram
+
+## Power
+ - watt
+
+## Pressure
+ - pascal
+
+## Resistance
+ - ohm
+
+## Temperature
+ - celsius
+ - farenheit
+ - kelvin
+
+## Time
+ - attosecond
+ - day
+ - exasecond
+ - femtosecond
+ - gigasecond
+ - hour
+ - kilosecond
+ - megasecond
+ - microsecond
+ - millisecond
+ - minute
+ - nanosecond
+ - petasecond
+ - picosecond
+ - second
+ - terasecond
+ - week
+ - yoctosecond
+ - yottasecond
+ - zeptosecond
+ - zettasecond
+
+## Velocity
+ - kilometerPerHour
+ - knot
+ - meterPerSecond
+ - milePerHour
+
+## Voltage
+ - volt
+
+## Volume
+ - cubicMeter
+ - litre
```

Comment:
It seems like the spelling "meter" is used over "metre" in this code, but here we use "litre" instead of "liter" which is a bit inconsistent. Both spellings are correct though, the "metre" and "litre" spelling are referred to as *international spelling* whereas "meter" and "liter" is *american spelling*. (see for instance https://en.wikipedia.org/wiki/Metre#Spelling). The "meter" spelling is also used in Swedish why it is more natural for me, whereas the "metre" spelling is the same as in French.

If it is not too much work, I would support both spelling conventions as they both seem to be in wide use.

Reply:
good idea. Added.